ecommerce/migrations/0006_cart_cartitem.py

```
@@ -0,0 +1,47 @@
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('ecommerce', '0005_product_productimage'),
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)